src/licensedcode/data/licenses/bsd-1-clause.LICENSE

```diff
@@ -0,0 +1,22 @@
+BSD 1 Clause License
+
+Copyright (c) [Copyright year]
+[Name of Copyright Owner]  All rights reserved.
```

Comment:
We never put copyrights in license texts, especially not templates

Reply:
Will take care.